render/win/wglext.h

```diff
@@ -501,10 +501,10 @@ typedef BOOL (WINAPI * PFNWGLCHOOSEPIXELFORMATARBPROC) (HDC hdc, const int *piAt
 #define WGL_ARB_make_current_read 1
 #ifdef WGL_WGLEXT_PROTOTYPES
 extern BOOL WINAPI wglMakeContextCurrentARB (HDC hDrawDC, HDC hReadDC, HGLRC hglrc);
-extern HDC WINAPI wglGetCurrentReadDCARB (void);
+extern HDC WINAPI wglGetCurrentReadDCARB ();
```

Comment:
`wglext.h` is an OpenGL header (for WIndows systems). It's not C++-specific, nor project-specific, so we probably shouldn't change it.

I'm actually a bit surprised it's included in the code base, but I guess MSVC doesn't have it (or it has an older version)?

Reply:
I was just looking at that a minute ago, too, and since it's extern it probably shouldn't be changed. Good call.

My `GLEW_INCLUDE_DIR` is in the AMD APP SDK and it doesn't have `wglext.h`, so I guess it needs to be in the codebase?